lib/ddtrace/contrib/grape/configuration/settings.rb

```diff
@@ -24,6 +25,13 @@ class Settings < Contrib::Configuration::Settings
           end
 
           option :service_name, default: Ext::SERVICE_NAME
+
+          option :error_statuses do |o|
+            o.default { Datadog::Contrib::StatusCodeMatcher.new(Datadog::Ext::HTTP::ERROR_RANGE.to_a) }
+            o.setter do |new_value, _old_value|
+              Datadog::Contrib::StatusCodeMatcher.new(new_value)
+            end
```

Comment:
Does the default case still work?

Won't it perform:
```ruby
# from o.default { Datadog::Contrib::StatusCodeMatcher.new(Datadog::Ext::HTTP::ERROR_RANGE.to_a) }
default = Datadog::Contrib::StatusCodeMatcher.new(Datadog::Ext::HTTP::ERROR_RANGE.to_a)

# from o.setter do |new_value, _old_value|
Datadog::Contrib::StatusCodeMatcher.new(default)
```

Which will likely break when `StatusCodeMatcher.new` receives an instance of `StatusCodeMatcher` instead of a string 🤔

Reply:
ah, woops, that's my bad i had suggested that

...yea let's use default as just `Datadog::Ext::HTTP::ERROR_RANGE.to_a`

Reply:
It doesn't break bc @Kyle-Neale wrote good error handling / checking in `StatusCodeMatcher` but, yea, we should be passing in the raw value, my fault.